src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig

```diff
@@ -75,20 +75,32 @@
 
         <h2>Rendered Templates</h2>
 
-        <table>
+        <table id="twig-table">
             <thead>
-                <tr>
-                    <th scope="col">Template Name</th>
-                    <th scope="col">Render Count</th>
-                </tr>
+            <tr>
+                <th scope="col">Template Name & Path</th>
+                <th class="num-col" scope="col">Render Count</th>
+            </tr>
             </thead>
             <tbody>
             {% for template, count in collector.templates %}
                 <tr>
                     {%- set file = collector.templatePaths[template]|default(false) -%}
                     {%- set link = file ? file|file_link(1) : false -%}
-                    <td>{% if link %}<a href="{{ link }}" title="{{ file }}">{{ template }}</a>{% else %}{{ template }}{% endif %}</td>
-                    <td class="font-normal">{{ count }}</td>
+                    <td>
+                        <span class="sf-icon icon-twig">{{ include('@WebProfiler/Icon/twig2.svg') }}</span>
+                        {% if link %}
+                            <a href="{{ link }}" title="{{ file }}">{{ template }}</a>
+                            <div>
+                                <a class="text-muted" href="{{ link }}" title="{{ file }}">
+                                    {{ collector.relativePaths[template]|default(file) }}
```

Comment:
what about moving relative path logic to twig bundle (CodeExtension), so here it be simply:

```
{{ file_relative(file) }}
```

Reply:
Done.